sections/03_categorize.md

```diff
@@ -142,6 +142,22 @@ This indicates a potential strength of deep methods. It may be possible to
 repurpose features from task to task, improving overall predictions as the field
 tackles new challenges.
 
+Several authors have created reusable feature sets for medical terminologies using
+neural embeddings, as popularized by word2Vec [@tag:Word2Vec]. This approach
+was first used on free text medical notes by De Vine et al.
+[@doi:10.1145/2661829.2661974] with results at or better than traditional methods.
+Y. Choi et al.[@doi:10.1145/2567948.2577348] built embeddings of standardized
+terminologies, such as ICD and NDC, used in widely available administrative
```

Comment:
Ok - when we go through and check acronyms + define at first occurrence I'll know now. Thanks! I think the DOI below is my only requested change.